spec/support/ansible_shared/automation_manager/job.rb

```diff
@@ -59,6 +59,7 @@
           expect(job.job_template).to          eq(template)
```

Comment:
@lfu Please review test failures.  Also, this file should be named `job_spec.rb`

cc @tinaafitz

Reply:
@gmcculloug Looks like files containing shared examples are named as regular ruby files. 
Check this [in core repo](https://github.com/ManageIQ/manageiq/tree/924a89012a1a68d0cda31e19e14cd49188d53a04/spec/support/examples_group) and [in Ansible Tower repo](https://github.com/ManageIQ/manageiq-providers-ansible_tower/tree/f9bdaeab214e2b9775d8ab736f4035307f0886f9/spec/support/ansible_shared).